src/krate.rs

```diff
@@ -1423,6 +1462,42 @@ pub fn owners(req: &mut Request) -> CargoResult<Response> {
     Ok(req.json(&R { users: owners }))
 }
 
+/// Handles the `GET /crates/:crate_id/owner_team` route.
+pub fn owner_team(req: &mut Request) -> CargoResult<Response> {
```

Comment:
What do you think about making this a single route and adding a `?kind` parameter if needed?